### PR TITLE
get-started freshness

### LINKED ISSUE #17841: Get Started -> Persist the DB -> See this in practice -> Step 2 : Sample code not working
### Is this a docs issue?

- [X] My issue is about the documentation content or website

### Type of issue

Information is incorrect

### Description

When I used the given code:
`docker exec <container-id> cat /data.txt`

 the file could not be found. Added a '.' to the beginning and value came up as expected.

image

### Location

https://docs.docker.com/get-started/05_persisting_data/#see-this-in-practice

### Suggestion

_No response_

### PR DESCRIPTION
**Pull request by craig-osterhout**
### Proposed changes

Refresh /get-started/

### Related issues (optional)

Fixes #17841 
Fixes hotjar#4613

### CODE REVIEW

File: get-started/02_our_app.md [hunk 1]
```diff
@@ -143,12 +136,12 @@ Now that you have an image, you can run the application in a [container](../get-
    ![Empty todo list](images/todo-list-empty.png){: style="width:450px;margin-top:20px;"}
    {: .text-center }
 
-3. Go ahead and add an item or two and see that it works as you expect. You can mark items as complete and remove them. Your frontend is successfully storing items in the backend.
+3.Add an item or two and see that it works as you expect. You can mark items as complete and remove them. Your frontend is successfully storing items in the backend.
```
Comment (dvdksn) [inline get-started/02_our_app.md:139]:
```suggestion
3. Add an item or two and see that it works as you expect. You can mark items as complete and remove them. Your frontend is successfully storing items in the backend.
```

File: get-started/04_sharing_app.md [hunk 2]
```diff
@@ -118,19 +111,24 @@ new instance that has never seen this container image. To do this, you will use
     > [Networking](../network/index.md#published-ports).
     {: .tip }
 
-6. Select on the 3000 badge when it comes up and you should see the app with your modifications.
-    If the 3000 badge doesn't show up, you can select on the **Open Port** button and type in 3000.
-
+6. Select the 3000 badge when it appears.
+    If the 3000 badge doesn't appear, you can select **Open Port** and specify `3000`.
```
Comment (dvdksn) [inline get-started/04_sharing_app.md:115]:
```suggestion
6. Select the 3000 badge when it appears.

   If the 3000 badge doesn't appear, you can select **Open Port** and specify `3000`.
```

File: get-started/02_our_app.md [hunk 3]
```diff
@@ -180,9 +173,16 @@ In Docker Desktop, select the **Containers** tab to see a list of your container
...
 </div>
 
-## Next steps
+## Summary
 
-In this short section, you learned the basics about creating a Dockerfile to build a container image. Once you built an image, you started a container and saw the running app.
+In this section, you learned the basics about creating a Dockerfile to build an image. Once you built an image, you started a container and saw the running app.
+
+Related information:
+ - [Dockerfile reference](../engine/reference/builder.md)
```
Comment (dvdksn) [inline get-started/02_our_app.md:181]:
```suggestion

 - [Dockerfile reference](../engine/reference/builder.md)
```

File: get-started/04_sharing_app.md [hunk 4]
```diff
@@ -118,19 +111,24 @@ new instance that has never seen this container image. To do this, you will use
...
+registry. You then went to a brand new instance and were able to run the freshly
+pushed image. This is quite common in CI pipelines, where the pipeline will
+create the image and push it to a registry and then the production environment
 can use the latest version of the image.
 
-Now you can circle back around to what you noticed at the end of the last
-section. As a reminder, you noticed that when you restarted the app, you lost all of your todo list items.
-That's obviously not a great user experience, so next you'll learn how you can persist the data across restarts.
+Related information:
+ - [docker CLI reference](/engine/reference/commandline/cli/)
```
Comment (dvdksn) [inline get-started/04_sharing_app.md:126]:
```suggestion

 - [docker CLI reference](/engine/reference/commandline/cli/)
```

File: get-started/05_persisting_data.md [hunk 5]
```diff
@@ -198,19 +202,18 @@ $ docker volume inspect todo-db
...
-> If you wanted to look at the actual contents of the mount point directory, you would need to look inside of
-> that VM.
+## Summary
 
-## Next steps
+In this section, you learned how to persist container data.
 
-At this point, you have a functioning application that can survive restarts.
+Related information:
+ - [docker CLI reference](/engine/reference/commandline/cli/)
```
Comment (dvdksn) [inline get-started/05_persisting_data.md:212]:
```suggestion

 - [docker CLI reference](/engine/reference/commandline/cli/)
```

File: get-started/06_bind_mounts.md [hunk 6]
```diff
@@ -309,14 +313,18 @@ Update your app on your host machine and see the changes reflected in the contai
...
 
 At this point, you can persist your database and see changes in your app as you develop without rebuilding the image.
 
 In addition to volume mounts and bind mounts, Docker also supports other mount
 types and storage drivers for handling more complex and specialized use cases.
-To learn more about the advanced storage concepts, see
-[Manage data in Docker](https://docs.docker.com/storage/).
+
+Related information:
+ - [docker CLI reference](/engine/reference/commandline/cli/)
```
Comment (dvdksn) [inline get-started/06_bind_mounts.md:324]:
```suggestion

 - [docker CLI reference](/engine/reference/commandline/cli/)
```

File: get-started/08_using_compose.md [hunk 7]
```diff
@@ -89,11 +75,11 @@ $ docker run -dp 127.0.0.1:3000:3000 \
...
    ```
- 
-4. Next, we'll migrate both the working directory (`-w /app`) and the volume mapping (`-v "$(pwd):/app"`) by using
-   the `working_dir` and `volumes` definitions. Volumes also has a [short](../compose/compose-file/05-services.md#short-syntax-5) and [long](../compose/compose-file/05-services.md#long-syntax-5) syntax.
 
-    One advantage of Docker Compose volume definitions is we can use relative paths from the current directory.
+4. Next, migrate both the working directory (`-w /app`) and the volume mapping
+   (`-v "$(pwd):/app"`) by using the `working_dir` and `volumes` definitions.
+
+    One advantage of Docker Compose volume definitions is you can use relative paths from the current directory.
```
Comment (dvdksn) [inline get-started/08_using_compose.md:82]:
```suggestion
   One advantage of Docker Compose volume definitions is you can use relative paths from the current directory.
```

File: get-started/08_using_compose.md [hunk 8]
```diff
@@ -220,30 +207,30 @@ volumes:
...
 
-    ```console
-    $ docker compose up -d
-    ```
+   ```console
+   $ docker compose up -d
+   ```
 
-    When we run this, we should see output like this:
+    When you run the previous command, you should see output like the following:
```
Comment (dvdksn) [inline get-started/08_using_compose.md:221]:
```suggestion
   When you run the previous command, you should see output like the following:
```

File: get-started/08_using_compose.md [hunk 9]
```diff
@@ -220,30 +207,30 @@ volumes:
...
-    ```
+   ```plaintext
+   Creating network "app_default" with the default driver
+   Creating volume "app_todo-mysql-data" with default driver
+   Creating app_app_1   ... done
+   Creating app_mysql_1 ... done
+   ```
 
-    You'll notice that the volume was created as well as a network! By default, Docker Compose automatically creates a network specifically for the application stack (which is why we didn't define one in the compose file).
+    You'll notice that Docker Compose created the volume as well as a network. By default, Docker Compose automatically creates a network specifically for the application stack (which is why you didn't define one in the Compose file).
```
Comment (dvdksn) [inline get-started/08_using_compose.md:230]:
```suggestion
   You'll notice that Docker Compose created the volume as well as a network. By default, Docker Compose automatically creates a network specifically for the application stack (which is why you didn't define one in the Compose file).
```

File: get-started/overview.md [hunk 10]
```diff
@@ -174,29 +173,28 @@ the default registry configuration):
...
 Docker uses a technology called `namespaces` to provide the isolated workspace
-called the *container*. When you run a container, Docker creates a set of
-*namespaces* for that container.
+called the container. When you run a container, Docker creates a set of
+namespaces for that container.
 
 These namespaces provide a layer of isolation. Each aspect of a container runs
 in a separate namespace and its access is limited to that namespace.
 
 ## Next steps
```
Comment (dvdksn) [inline get-started/overview.md:198]: Is there a newline after the heading? hard to tell from the GH review UI but it looks like we need to add one

```suggestion
## Next steps

```

File: get-started/06_bind_mounts.md [hunk 11]
```diff
@@ -309,14 +313,19 @@ Update your app on your host machine and see the changes reflected in the contai
...
 
 In addition to volume mounts and bind mounts, Docker also supports other mount
 types and storage drivers for handling more complex and specialized use cases.
-To learn more about the advanced storage concepts, see
-[Manage data in Docker](https://docs.docker.com/storage/).
+
+Related information:
+
+ - [docker CLI reference](/engine/reference/commandline/cli/)
+ - [Manage data in Docker](https://docs.docker.com/storage/).
```
Comment (craig-osterhout) [inline get-started/06_bind_mounts.md:326]:
```suggestion
 - [Manage data in Docker](https://docs.docker.com/storage/)
```

**Review by dvdksn — APPROVED** [review]
!

**Pull request merged**